### PR TITLE
Implement hw 3

### PR DESCRIPTION
**Pull request by yanezh**
Implement hw's 3 and 2-2

### CODE REVIEW

File: 2015/SuvitovEvgeniy/3/lib/Output.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Output
+  def initialize(review)
+    @review = review
+    @words ||= YAML.load_file('../lib/word.yml')
+  end
+
+  def print
```
Comment (yanezh) [inline 2015/SuvitovEvgeniy/3/lib/Output.rb]: Я уж не знаю, что тут ещё упростить можно.

File: 2015/SuvitovEvgeniy/3/bin/bsuir_reviews [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'colorize'
+require 'mechanize'
+require 'pry'
+require 'yaml'
+Dir['../lib/*.rb'].each { |x| require_relative x }
+options = ArgumentParser.new.parse
```
Comment (Nazarchuk) [inline 2015/SuvitovEvgeniy/3/bin/bsuir_reviews]: Not OOP

File: 2015/SuvitovEvgeniy/3/bin/bsuir_reviews [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+if options[:help]
+  puts 'Print "./bsuir_reviews [number of group]" to get reviews about teachers of this group'.red
```
Comment (Nazarchuk) [inline 2015/SuvitovEvgeniy/3/bin/bsuir_reviews]: Replace magic strings on constants

File: 2015/SuvitovEvgeniy/3/lib/ReviewParser.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    @first_letter_patron = string.split[2][0]
+    @reviews = []
+    @dates = []
+  end
+
+  def parse
+    agent = Mechanize.new
+    @@page ||= agent.get('http://bsuir-helper.ru/lectors')
+    @@page.links.each do |link|
+      next if !link.text.include?(@surname) ||
```
Comment (Nazarchuk) [inline 2015/SuvitovEvgeniy/3/lib/ReviewParser.rb]: sipmly unless

File: 2015/SuvitovEvgeniy/3/lib/ReviewParser.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    @surname = string.split[0] + ' '
+    @first_letter_name = string.split[1][0]
+    @first_letter_patron = string.split[2][0]
+    @reviews = []
+    @dates = []
+  end
+
+  def parse
+    agent = Mechanize.new
+    @@page ||= agent.get('http://bsuir-helper.ru/lectors')
```
Comment (Nazarchuk) [inline 2015/SuvitovEvgeniy/3/lib/ReviewParser.rb]: Rise up this under class definition line or don't use class variables at all.
https://github.com/bbatsov/ruby-style-guide#no-class-vars

File: 2015/SuvitovEvgeniy/3/lib/SurnameParser.rb [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  def parse
+    agent = Mechanize.new
+    page = agent.get("http://www.bsuir.by/schedule/schedule.xhtml?id=#{number}")
+    page.links.each do |link|
+      list_of_surnames << link.text.to_s
+    end
+    list_of_surnames
+  rescue
+    p 'No Internet'
+    Kernel.abort
```
Comment (Nazarchuk) [inline 2015/SuvitovEvgeniy/3/lib/SurnameParser.rb]: Your class Kernel ancestor, simply abort

**Pull request merged**